backend/app/api/endpoints/runtime_work.py

```
@@ -587,6 +590,64 @@ async def create_runtime_task_endpoint(
+    if not backend_url:
+        scheme = fastapi_request.url.scheme
+        host = fastapi_request.headers.get("host", fastapi_request.url.netloc)
+        backend_url = f"{scheme}://{host}"
+    return f"{backend_url.rstrip('/')}{settings.API_PREFIX}/runtime-work"
+
+
+@router.post("/llm-responses-proxy/{token}/responses")
+async def llm_responses_proxy_endpoint(
+    token: str,
+    fastapi_request: Request,
+    db: Session = Depends(get_db),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)